react/typings/react-intl.d.ts

```diff
@@ -0,0 +1,7 @@
+import { MessageDescriptor as StandardMessageDescriptor } from 'react-intl'
+
+declare module 'react-intl' {
+  export interface MessageDescriptor extends StandardMessageDescriptor {
```

Comment:
(We should create our own npm package to avoid writing this everywhere)

Reply:
We will have the lib builder soon